libs/icons/atom.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="#FFF" viewBox="0 0 76.314 69.686">
+  <path d="M42.71 33.832c.5 2.4-1 4.8-3.4 5.3s-4.7-1-5.2-3.4 1-4.8 3.4-5.3c2.3-.5 4.7 1 5.2 3.4M24.38.002c-1.143.021-2.265.27-3.306.807-2.447 1.232-3.934 3.663-4.646 6.421-.715 2.767-.776 6.003-.352 9.63.134 1.145.404 2.391.637 3.61-1.399.313-2.809.622-4.068 1.012-3.486 1.08-6.418 2.437-8.655 4.2-2.226 1.754-3.846 4.085-3.972 6.81-.143 2.368.863 4.594 2.433 6.395 1.574 1.804 3.718 3.315 6.305 4.662a2.5 2.5 0 1 0 2.308-4.434c-2.213-1.153-3.869-2.393-4.845-3.513-.977-1.12-1.265-1.988-1.213-2.815v-.025l.002-.028c.037-.908.559-1.928 2.078-3.125 1.52-1.197 3.95-2.394 7.039-3.351 1.133-.351 2.438-.634 3.734-.92a79.368 79.368 0 0 0 2.575 7.72c-3.267 6.043-5.592 11.913-6.774 17.065-.814 3.55-1.116 6.764-.736 9.58.38 2.817 1.533 5.398 3.799 6.908l.025.018.023.016c2.234 1.414 4.973 1.493 7.618.79 2.645-.702 5.361-2.152 8.177-4.171a2.5 2.5 0 1 0-2.914-4.063c-2.483 1.781-4.767 2.93-6.547 3.403-1.76.467-2.86.3-3.626-.174-.726-.495-1.342-1.479-1.6-3.395-.26-1.933-.069-4.644.654-7.795.825-3.595 2.394-7.783 4.496-12.181.459.972.855 1.933 1.356 2.912v.002l.002.002c4.612 8.969 10.1 16.44 15.465 21.394 2.682 2.477 5.328 4.34 7.972 5.405 2.645 1.065 5.478 1.326 7.918.087 2.366-1.201 3.815-3.543 4.53-6.195.714-2.652.81-5.747.45-9.217-.717-6.939-3.31-15.396-7.566-24.113a2.5 2.5 0 1 0-4.492 2.193c4.044 8.283 6.452 16.325 7.084 22.436.316 3.055.175 5.62-.305 7.4-.479 1.78-1.18 2.639-1.964 3.037-.81.411-1.977.461-3.786-.267-1.808-.729-4.068-2.242-6.449-4.44-4.76-4.395-10.021-11.472-14.408-20.002l-.002-.002v-.002a92.343 92.343 0 0 1-2.883-6.203c.873-1.525 1.751-3.054 2.752-4.588a95.149 95.149 0 0 1 3.639-5.156 93.236 93.236 0 0 1 6.34.086c9.55.487 18.148 2.389 24.107 4.94 2.98 1.275 5.288 2.723 6.701 4.083 1.38 1.327 1.818 2.401 1.805 3.303-.198 1.463-2.293 3.849-6.813 5.656a2.5 2.5 0 1 0 1.856 4.643c5.416-2.166 9.504-5.33 9.959-9.873l.008-.072.004-.073c.112-2.734-1.294-5.207-3.352-7.187-2.058-1.98-4.843-3.639-8.201-5.076-6.716-2.875-15.769-4.823-25.819-5.336-.886-.045-1.723-.015-2.595-.037 2.97-3.409 5.948-6.295 8.71-8.405 2.592-1.978 4.997-3.296 6.874-3.873 1.855-.57 2.994-.42 3.728.024.824.563 1.379 1.43 1.668 3.072.294 1.666.198 4.02-.363 6.824a2.5 2.5 0 1 0 4.902.98c.64-3.196.841-6.089.385-8.673-.456-2.584-1.694-4.96-3.85-6.397l-.03-.021-.034-.02C56.015.867 53.18.906 50.457 1.742s-5.53 2.457-8.44 4.678c-4.002 3.057-8.176 7.341-12.183 12.414-2.88.121-5.604.386-8.201.76-.218-1.127-.47-2.283-.59-3.315-.376-3.21-.261-5.918.225-7.799.485-1.88 1.219-2.792 2.06-3.212l.016-.008.017-.008c.611-.317 1.493-.407 2.946.06 1.452.468 3.326 1.515 5.375 3.098a2.5 2.5 0 1 0 3.056-3.957c-2.351-1.817-4.625-3.168-6.898-3.9-1.136-.366-2.31-.572-3.46-.551zm1.599 24.086c-.485.703-.99 1.351-1.463 2.078-.295.452-.528.9-.813 1.352-.35-1.05-.6-2.03-.887-3.04 1.041-.141 2.061-.289 3.163-.39z"/>
```

Comment:
Nice, and [svgomg](https://jakearchibald.github.io/svgomg/) can shrink it 1.46K => 838 bytes:

```xml
<svg xmlns="http://www.w3.org/2000/svg" fill="#FFF" viewBox="0 0 76.3 69.7">
  <path d="M42.7 33.8c.5 2.4-1 4.8-3.4 5.3s-4.7-1-5.2-3.4 1-4.8 3.4-5.3c2.3-.5 4.7 1 5.2 3.4M24.4 0c-1.2 0-2.3.3-3.3.8a9.5 9.5 0 0 0-4.7 6.4c-.7 2.8-.7 6-.3 9.7l.6 3.6-4 1c-3.5 1-6.5 2.4-8.7 4.2a9.4 9.4 0 0 0-4 6.8 9 9 0 0 0 2.5 6.4c1.5 1.8 3.7 3.3 6.3 4.6a2.5 2.5 0 1 0 2.3-4.4 17 17 0 0 1-4.9-3.5c-1-1.1-1.2-2-1.2-2.8 0-1 .6-2 2-3.2 1.6-1.2 4-2.4 7.1-3.3l3.8-1a79.4 79.4 0 0 0 2.5 7.8c-3.2 6-5.6 11.9-6.7 17-.9 3.6-1.2 6.8-.8 9.6.4 2.8 1.6 5.4 3.8 7 2.3 1.4 5 1.4 7.7.7 2.6-.7 5.3-2.1 8.2-4.1a2.5 2.5 0 1 0-3-4.1c-2.4 1.8-4.7 3-6.5 3.4-1.8.5-2.9.3-3.6-.2-.7-.5-1.4-1.4-1.6-3.4-.3-1.9 0-4.6.6-7.8A61 61 0 0 1 23 39.1l1.4 2.9c4.6 9 10 16.4 15.5 21.4a27 27 0 0 0 8 5.4c2.6 1 5.4 1.3 7.8 0a9.4 9.4 0 0 0 4.6-6.1c.7-2.7.8-5.8.4-9.3-.7-6.9-3.3-15.3-7.5-24a2.5 2.5 0 1 0-4.5 2.1c4 8.3 6.4 16.4 7 22.5.4 3 .2 5.6-.3 7.4-.4 1.7-1.1 2.6-2 3-.7.4-1.9.5-3.7-.3-1.8-.7-4-2.2-6.5-4.4a74.3 74.3 0 0 1-14.4-20 92.3 92.3 0 0 1-2.8-6.2l2.7-4.6a95.1 95.1 0 0 1 3.6-5.2 93.2 93.2 0 0 1 6.4.1c9.5.5 18.1 2.4 24 5a23 23 0 0 1 6.8 4c1.4 1.4 1.8 2.5 1.8 3.4-.2 1.4-2.3 3.8-6.8 5.6a2.5 2.5 0 1 0 1.8 4.7c5.5-2.2 9.5-5.4 10-10a9.6 9.6 0 0 0-3.3-7.3c-2.1-2-4.9-3.6-8.2-5a77.6 77.6 0 0 0-25.9-5.4h-2.6c3-3.4 6-6.3 8.8-8.4 2.5-2 5-3.3 6.8-3.9 1.9-.5 3-.4 3.8 0 .8.6 1.3 1.5 1.6 3.1.3 1.7.2 4-.3 6.8a2.5 2.5 0 1 0 4.9 1c.6-3.2.8-6 .3-8.7-.4-2.5-1.6-5-3.8-6.3-2.4-1.5-5.2-1.5-8-.7S45 4.2 42 6.4c-4 3-8.2 7.4-12.2 12.4-2.8.2-5.6.4-8.2.8l-.6-3.3c-.3-3.2-.2-6 .3-7.8.5-1.9 1.2-2.8 2-3.2.7-.4 1.6-.5 3 0a19 19 0 0 1 5.4 3.1 2.5 2.5 0 1 0 3-4A23.2 23.2 0 0 0 27.8.7C26.7.2 25.5 0 24.4 0zM26 24l-1.5 2.2-.8 1.3-.9-3L26 24z"/>
</svg>
```